import java.util.*;
class exa
{
  public static void main(String []arga)
  {
        throw new ArithmeticException("divisible by Zero");
  }
}                                                                      //throw keyword is used create exception object explicitly and handover to the JVM manually.
                                                                      
